File: TesminePoulose_COMP304Sec001_Lab2_Ex1/app/src/main/java/com/example/tesminepoulose_comp304sec001_lab2_ex1/MainActivity.java
package com.example.tesminepoulose_comp304sec001_lab2_ex1;

import androidx.appcompat.app.AppCompatActivity;

import android.content.Intent;
import android.os.Bundle;
import android.view.View;
import android.widget.Button;

public class MainActivity extends AppCompatActivity {
    public Button button;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        System.out.println("before Moving to Menu");
        setContentView(R.layout.activity_main);
        button = (Button) findViewById(R.id.openMusic);
        button.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View view) {

                Intent intent = new Intent(MainActivity.this, MusicMenuPage.class);
                System.out.println("Moving to Menu");
                startActivity(intent);

            }
        });
    }
}

File: GokulrajVenugopa_COMP304Lab1_Ex1/app/src/main/java/com/example/gokulrajvenugopa_comp304lab1_ex1/MainActivity.kt
package com.example.gokulrajvenugopa_comp304lab1_ex1

import android.content.Intent
import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import android.util.Log
import android.widget.TextView
import com.example.gokulrajvenugopa_comp304lab1_ex1.databinding.ActivityMainBinding

class MainActivity : AppCompatActivity() {

    companion object {
        private const val TAG = "MainActivity"
    }
    private lateinit var binding: ActivityMainBinding

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        binding = ActivityMainBinding.inflate(layoutInflater)
        setContentView(binding.root)
        //setContentView(R.layout.activity_main)
        setTitle("My First App");



        binding.send.setOnClickListener {
            val intent = Intent(this, SecondActivity::class.java)
            val textView = binding.nameEditTextViewFirstActivity.text

           Log.e(TAG,textView.toString())
            intent.putExtra("DATA", textView.toString() )
            startActivity(intent)
        }
    }
}